script/tool/test/firebase_test_lab_command_test.dart

```diff
@@ -203,12 +282,87 @@ void main() {
       );
     });
 
-    test('skips packages with no androidTest directory', () async {
+    test('fails for packages with no androidTest directory', () async {
       createFakePlugin('plugin', packagesDir, extraFiles: <String>[
         'example/integration_test/foo_test.dart',
         'example/android/gradlew',
       ]);
 
+      Error? commandError;
+      final List<String> output = await runCapturingPrint(
+        runner,
+        <String>[
+          'firebase-test-lab',
+          '--device',
+          'model=flame,version=29',
+          '--device',
+          'model=seoul,version=26',
+          '--test-run-id',
+          'testRunId',
+          '--build-id',
+          'buildId',
+        ],
+        errorHandler: (Error e) {
+          commandError = e;
+        },
+      );
+
+      expect(commandError, isA<ToolExit>());
+      expect(
+        output,
+        containsAllInOrder(<Matcher>[
+          contains('Running for plugin'),
+          contains('No androidTest directory found.'),
+          contains('The following packages had errors:'),
+          contains('plugin:\n'
+              '    No tests ran (use --exclude if this is intentional).'),
```

Comment:
nit: does it make sense to say where `--exclude` should be used? e.g. in .cirrus.yml?

Reply:
It can be useful locally as well; in any scenario where someone sees this message, they are trying to run a failing package, so might want to exclude it.